packages/schematics/angular/application/schema.json

```diff
@@ -19,57 +20,57 @@
       "x-prompt": "What name would you like to use for the application?"
     },
     "experimentalIvy": {
-      "description": "EXPERIMENTAL: Specifies whether to create a new application which uses the Ivy rendering engine.",
+      "description": "EXPERIMENTAL: True to create a new app that uses the Ivy rendering engine.",
       "type": "boolean",
       "default": false
     },
     "inlineStyle": {
-      "description": "Specifies if the style will be in the ts file.",
+      "description": "When true, includes styles inline in the root component TS file. Only CSS styles can be included inline. Default is false, meaning that an external styles file is created and referenced in the root component TS file.",
```

Comment:
Maybe `.ts` file? Maybe `component.ts` file? I looked at some of the other doc for template syntax and components. They use just "component file" or a filename, but I like where you're headed clarifying that it's the typescript file